source/files/stylesheets/adjustments.css

```diff
@@ -487,18 +343,12 @@ html:not(.csscalc) .layout-footer .footer-row-1 > .fg-col:last-child {
   height: auto;
 }
 
-.layout-footer .footer-row-1 .social-icon {
-  float: left;
```

Comment:
Removing this fixes the social media icons on IE 11, but changes their spacing on other browsers.